src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractLocalFunctionTests.cs

```diff
@@ -3790,6 +3791,581 @@ static int NewMethod()
 }", CodeActionIndex);
         }
 
+        [Fact, WorkItem(40188, "https://github.com/dotnet/roslyn/issues/40188"), Trait(Traits.Feature, Traits.Features.CodeActionsExtractLocalFunction)]
+        public async Task TestEditorconfigSetting_StaticLocalFunction_True()
+        {
+            var input = @"
+<Workspace>
```

Comment:
nit: In the future try to separate out the code and editorconfig files. It's hard to read them bunched together, especially since the editorconfig doesn't change :)